packages/pwa/app/utils/locale.test.js

```diff
@@ -22,7 +22,7 @@ const nonSupportedLocale = 'nl-NL'
 const supportedLocale = supportedLocales[1]
 
 const testId1 = 'footer.link.privacy_policy'
-const testId2 = 'homepage.message.welcome'
+const testId2 = 'account.accordion.button.my_account'
```

Comment:
`homepage.message.welcome` was just some sample id we used. Now that we have real translations, we can use real message ids too.